toolkit/tools/pkg/imagecustomizerlib/imagecustomizer_test.go

```diff
@@ -44,7 +44,7 @@ func TestCustomizeImageEmptyConfig(t *testing.T) {
 
 	// Customize image.
 	err = CustomizeImage(buildDir, buildDir, &imagecustomizerapi.Config{}, diskFilePath, nil, outImageFilePath,
-		"vhd", false)
+		"vhd", "raw", false)
```

Comment:
Why not just use "" here instead of "raw"? (Same thing below.)

Reply:
Yes, I was thinking of that too - updated